code/modules/economy/holopay.dm

```diff
@@ -1,3 +1,6 @@
+/* LETHAL EDIT: disables holopay
```

Comment:
We can't delete the holopay stand structure because its present on some maps.
Make a module in our modular files and make them delete on spawn if you really really want them gone.